<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>Single-cell integration benchmarking</title>

<script src="site_libs/header-attrs-2.3/header-attrs.js"></script>
<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/yeti.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 45px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 50px;
  margin-top: -50px;
}
.section h2 {
  padding-top: 50px;
  margin-top: -50px;
}
.section h3 {
  padding-top: 50px;
  margin-top: -50px;
}
.section h4 {
  padding-top: 50px;
  margin-top: -50px;
}
.section h5 {
  padding-top: 50px;
  margin-top: -50px;
}
.section h6 {
  padding-top: 50px;
  margin-top: -50px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">



<div id="nav-placeholder"></div>

<script>
$(function(){
  $("#nav-placeholder").load("navbar-content.html");
});
</script>

<div class="fluid-row" id="header">



<h1 class="title toc-ignore">Single-cell integration benchmarking</h1>

</div>


<p>Single-cell integration benchmarking (scIB) is a project to assess the performance of scRNA-seq batch integration methods. We have used 14 metrics to evaluate <strong>17 methods</strong> on <strong>7 scRNA-seq</strong> (5 real and 2 simulated) and <strong>6 scATAC-seq</strong> tasks. These metrics are designed to test both <strong>batch correction</strong> and <strong>conservation of biological variance</strong>.</p>
<p>For each task we also consider different combinations of pre-processing, including highly variable gene (HVG) selection and scaling. The usability and scalability of methods are also assessed.</p>
<p><img src="figures/index/overview.png" width="80%" style="display: block; margin: auto;" /></p>
<div id="results" class="section level1">
<h1>Results</h1>
<p>Here you can view all the individual benchmarking metrics broken down by dataset and method.</p>
<div class="row">
<div id="datasets" class="section level2 col-sm-6 col-md-3">
<h2>Datasets</h2>
<ul>
<li><a href="dataset_immune_cell_hum.html">immune_cell_hum</a></li>
<li><a href="dataset_immune_cell_hum_mou.html">immune_cell_hum_mou</a></li>
<li><a href="dataset_lung_atlas.html">lung_atlas</a></li>
<li><a href="dataset_mouse_brain.html">mouse_brain</a></li>
<li><a href="dataset_pancreas.html">pancreas</a></li>
<li><a href="dataset_simulations_1_1.html">simulations_1_1</a></li>
<li><a href="dataset_simulations_2.html">simulations_2</a></li>
</ul>
</div>
<div id="methods" class="section level2 col-sm-6 col-md-3">
<h2>Methods</h2>
<ul>
<li><a href="method_BBKNN.html">BBKNN</a></li>
<li><a href="method_ComBat.html">ComBat</a></li>
<li><a href="method_Conos.html">Conos</a></li>
<li><a href="method_DESC.html">DESC</a></li>
<li><a href="method_FastMNN.html">FastMNN</a></li>
<li><a href="method_Harmony.html">Harmony</a></li>
<li><a href="method_LIGER.html">LIGER</a></li>
<li><a href="method_MNN.html">MNN</a></li>
<li><a href="method_SAUCIE.html">SAUCIE</a></li>
<li><a href="method_Scanorama.html">Scanorama</a></li>
<li><a href="method_scANVI*.html">scANVI*</a></li>
<li><a href="method_scGen*.html">scGen*</a></li>
<li><a href="method_scVI.html">scVI</a></li>
<li><a href="method_Seurat%20v3%20CCA.html">Seurat v3 CCA</a></li>
<li><a href="method_Seurat%20v3%20RPCA.html">Seurat v3 RPCA</a></li>
<li><a href="method_trVAE.html">trVAE</a></li>
<li><a href="method_Unintegrated.html">Unintegrated</a></li>
</ul>
</div>
<div id="atac-datasets" class="section level2 col-sm-6 col-md-3">
<h2>ATAC Datasets</h2>
<ul>
<li><a href="dataset_atac_mouse_brain_atac_genes_large.html">genes_large</a></li>
<li><a href="dataset_atac_mouse_brain_atac_genes_small.html">genes_small</a></li>
<li><a href="dataset_atac_mouse_brain_atac_peaks_large.html">peaks_large</a></li>
<li><a href="dataset_atac_mouse_brain_atac_peaks_small.html">peaks_small</a></li>
<li><a href="dataset_atac_mouse_brain_atac_windows_large.html">windows_large</a></li>
<li><a href="dataset_atac_mouse_brain_atac_windows_small.html">windows_small</a></li>
</ul>
</div>
<div id="atac-methods" class="section level2 col-sm-6 col-md-3">
<h2>ATAC Methods</h2>
<ul>
<li><a href="method_BBKNN.html">BBKNN</a></li>
<li><a href="method_ComBat.html">ComBat</a></li>
<li><a href="method_Conos.html">Conos</a></li>
<li><a href="method_DESC.html">DESC</a></li>
<li><a href="method_FastMNN.html">FastMNN</a></li>
<li><a href="method_Harmony.html">Harmony</a></li>
<li><a href="method_LIGER.html">LIGER</a></li>
<li><a href="method_MNN.html">MNN</a></li>
<li><a href="method_SAUCIE.html">SAUCIE</a></li>
<li><a href="method_Scanorama.html">Scanorama</a></li>
<li><a href="method_scANVI*.html">scANVI*</a></li>
<li><a href="method_scGen*.html">scGen*</a></li>
<li><a href="method_scVI.html">scVI</a></li>
<li><a href="method_Seurat%20v3%20CCA.html">Seurat v3 CCA</a></li>
<li><a href="method_Seurat%20v3%20RPCA.html">Seurat v3 RPCA</a></li>
<li><a href="method_trVAE.html">trVAE</a></li>
<li><a href="method_Unintegrated.html">Unintegrated</a></li>
</ul>
</div>
</div>
<div id="usability" class="section level2">
<h2>Usability</h2>
<p>You can view the usability scores for each method <a href="usability.html">here</a>.</p>
</div>
</div>
<div id="links" class="section level1">
<h1>Links</h1>
<p>More information about the project can be found at these links:</p>
<ul>
<li><strong><a href="https://github.com/theislab/scib" title="scIB package GitHub repository">scIB package</a></strong> - Python module implementing the metrics and wrapping the methods used in the evaluation.</li>
<li><strong><a href="https://github.com/theislab/scib-pipeline" title="scIB pipeline GitHub repository">scIB pipeline</a></strong> - Snakemake pipeline implementing the evaluation workflow.</li>
<li><strong><a href="https://github.com/theislab/scib-reproducibility" title="scIB reproducibility GitHub repository">scIB reproducibility</a></strong> - Additional code used to produce the results in the paper (including this website).</li>
</ul>
</div>
<div id="citation" class="section level1">
<h1>Citation</h1>
<p>If any part of the project is useful for your work please cite:</p>
<blockquote>
<p>Luecken MD, Buttner M, Chaichoompu K, Danese A, Interlandi M, Mueller MF, Strobl DC, Zappia L, Dugas M, Colome-Tatche M, Theis FJ. <em>“Benchmarking atlas-level data integration in single-cell genomics”</em> <strong>bioRxiv</strong>. 2020 DOI: <a href="https://doi.org/10.1101/2020.05.22.111161" title="scIB preprint">10.1101/2020.05.22.111161</a></p>
</blockquote>
</div>




</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
